snmp/metadata.csv

```diff
@@ -787,6 +787,6 @@ snmp.powerSupplyStatus,gauge,,,,[Dell Poweredge] The status of the power supply.
 snmp.processorDeviceThreadCount,gauge,,thread,,[Dell Poweredge] The number of processor threads detected for the processor device.,0,snmp,
 snmp.batteryStatus,gauge,,,,[Dell Poweredge] The status of the battery.,0,snmp,
 snmp.batteryReading,gauge,,,,[Dell Poweredge] The reading of the battery.,0,snmp,
-datadog.snmp.check_duration,gauge,,second,,The duration of a check run in seconds.,0,snmp,
-datadog.snmp.check_interval,count,,second,,The interval between check runs in seconds.,0,snmp,
-datadog.snmp.submitted_metrics,gauge,,,,The number of submitted metrics for a check run.,0,snmp,
+datadog.snmp.check_duration,gauge,,second,,"The duration of a check run in seconds. The time needed for the integration to run once, that includes time to collect snmp data from devices, processing and submitting metrics/service checks/etc.",0,snmp,
+datadog.snmp.check_interval,count,,second,,The interval between check runs in seconds. The time delta between end of current check run and end of last check run,0,snmp,
```

Comment:
Mixed feelings on this one. We actually send the epoch of check end time, and its metrics that will convert it into the delta between samples as we send it as `monotonic_count`

Reply:
@pducolin 
From the integration point of view, we send timestamp as `monotonic_count`.
From the in-app/backend/UI point of view, it's a time delta as `count`.

The type and description in `metadata.csv` is about the in-app point of view, e.g. the type for `datadog.snmp.check_interval` is `count`.